### ISSUE TITLE
It is unreasonable that the 'chromium_arg' parameter must be provided as a comma-separated list.

### ISSUE THREAD

**Issue by WinXsp** [seleniumbase/SeleniumBase#3139]
i can't use
--window-size=100,100
--window-position=100,100
and 
--app=data:,

If using the following function will cause the window to flash.:
set_window_position(100,100)
set_window_size(100,100)

How to solve this problem?

**Comment by mdmintz** [comment]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2872#issuecomment-2190290905.

----

Window size and position must be set ***after* the browser has been launched**. Eg:

```python
driver.set_window_size(1000, 800)

driver.set_window_position(500, 400)

driver.set_window_rect(500, 400, 1000, 800) 
```

Related methods from [method_summary.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/method_summary.md):

https://github.com/seleniumbase/SeleniumBase/blob/2aa5abe9b84f633fc5a7610e0c735b4d806d6769/help_docs/method_summary.md?plain=1#L238

https://github.com/seleniumbase/SeleniumBase/blob/2aa5abe9b84f633fc5a7610e0c735b4d806d6769/help_docs/method_summary.md?plain=1#L240

https://github.com/seleniumbase/SeleniumBase/blob/2aa5abe9b84f633fc5a7610e0c735b4d806d6769/help_docs/method_summary.md?plain=1#L236

----

`chromium_arg` takes a comma-separated list of args, which means that individual args cannot contain commas.
(Changing that would break backwards-compatibility.)

**Comment by WinXsp** [comment]
Is it possible to add extra parameters to implement set_window_rect before startup without breaking backward compatibility?

**Comment by mdmintz** [comment]
I'll add the ability to pass in a list to `chromium_arg` in the next release: https://github.com/seleniumbase/SeleniumBase/issues/3144

**Comment by mdmintz** [comment]
This was resolved in `4.31.0` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.31.0
